pymc/model_graph.py

```diff
@@ -41,6 +43,118 @@ def fast_eval(var):
     return function([], var, mode="FAST_COMPILE")()
 
 
+class NodeType(str, Enum):
+    """Enum for the types of nodes in the graph."""
+
+    POTENTIAL = "Potential"
+    BASIC_RV = "Basic Random Variable"
```

Comment:
model.basic_RVs contains both observed and free RVs

```suggestion
    FREE_RV = "Basic Random Variable"
```

Reply:
Should I change the name of the string as well?

Reply:
Slash do all of the string names make sense?

Reply:
> Should I change the name of the string as well?

Yeah

And yes the others made sense